crates/polars-lazy/src/physical_plan/file_cache.rs

```diff
@@ -50,10 +50,11 @@ impl FileCache {
     where
         F: FnMut() -> PolarsResult<DataFrame>,
     {
+        #[cfg(debug_assertions)]
```

Comment:
We can use `debug_assert_ne`, which is more concise.

Reply:
Done, thanks